Library/Formula/relx.rb

```diff
@@ -0,0 +1,43 @@
+class Relx < Formula
+  homepage "https://github.com/erlware/relx"
+  url "https://github.com/erlware/relx.git", :tag => "v1.2.0"
+
+  head "https://github.com/erlware/relx.git"
+
+  depends_on "erlang"
+  depends_on "rebar"
+
+  def install
+    system "make", "escript"
+    bin.install "relx"
```

Comment:
That was actually my first approach, the problem is that relx, upon build, uses the git tag command to obtain the version to output with relx --version, if we use the .tar.gz there is no git repo so the output to the version command will be full of errors with no useful information.

Rebar offers a plugin that does just that, extracts the current tag and replaces a placeholder in a config file, it's a usual procedure in erlang escriptized projects, is the use of a git url not acceptable in this case?